test/test_torch.py

```diff
@@ -2092,6 +2092,37 @@ def test_gesv(self):
         torch.gesv(b, a, out=(tb, ta))[0]
         self.assertEqual(res1, tb)
 
+    def _test_bgesv(self, cast):
+        # test against gesv
+        A = cast(torch.randn(1, 5, 5))
+        b = cast(torch.randn(1, 5, 10))
+        x_exp, LU_exp = torch.gesv(b.squeeze(0), A.squeeze(0))
+        x, LU = torch.bgesv(b, A)
+        self.assertEqual(x, x_exp.unsqueeze(0))
+        self.assertEqual(LU, LU_exp.unsqueeze(0))
```

Comment:
Can you make a test that compares to `gesv` in a short loop (let's say we have a 4 elem batch)?